# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/tcp_wrappers/hotfix.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- tcp_wrappers_7.6/scaffold.c.vanilla	2022-05-21 14:03:05.506585352 +0200
+++ tcp_wrappers_7.6/scaffold.c	2022-05-21 14:03:14.094585010 +0200
@@ -26,10 +26,6 @@
 #define	INADDR_NONE	(-1)		/* XXX should be 0xffffffff */
 #endif
 
-#ifndef INET6
-extern char *malloc();
-#endif
-
 /* Application-specific. */
 
 #include "tcpd.h"
--- tcp_wrappers_7.6/myvsyslog.c.vanilla	2022-05-21 14:01:43.960588604 +0200
+++ tcp_wrappers_7.6/myvsyslog.c	2022-05-21 14:01:55.278588153 +0200
@@ -14,6 +14,7 @@
 #ifdef vsyslog
 
 #include <stdio.h>
+#include <syslog.h>
 
 #include "tcpd.h"
 #include "mystdarg.h"
